api/src/entities/Multiflash.py

```diff
@@ -48,11 +51,11 @@ def validate_composition(cls, v, values):
         return v
 
     def compute(self) -> MultiflashResult:
-        ph_index, ph_frac, moles = libhg.mf(
-            nc=self.number_of_components,
-            list=self.component_ids,
-            t=self.temperature,
-            p=self.pressure,
-            compos=self.composition,
+        phase_label, phase_fraction, moles = libhg.compute_multiflash(
+            num_comp=self.number_of_components,
```

Comment:
```suggestion
            num_comp=len(self.components_ids),
```
Just do this already?

Reply:
Thought about it, but couldn't decide if number_of_components should be a field in Multiflash anymore. If not, then the validation need to be rewritten. Thought it would be OK to do it in a new PR.